pkg/printers/printers_test.go

```diff
@@ -26,9 +26,18 @@ func TestTablePrinter(t *testing.T) {
 		},
 	}
 	printer := NewTablePrinter()
-	printer.PrintObj(test, os.Stdout)
+
+	printer.(*TablePrinter).AddColumn("CLUSTERNAME", func(c *eks.Cluster) string {
```

Comment:
Perhaps just `NAME`, as argument name is `--name`.

Reply:
I agree. It was calld `NAME` when i initially made the change. So i'll change it back.